Update learn-nft-projects.md. Current active projects. (#6540)
w3f/polkadot-wiki: docs/learn/learn-nft-projects.md

<!-- prettier-ignore -->
<!-- no toc -->

- [Asset Hub](#asset-hub)
- [Astar](#astar)
- [Basilisk](#basilisk)
- [Crust](#crust)
- [Darwinia Network](#darwinia-network)
- [KodaDot](#kodadot)
- [MNet Continuum](#mnet-continuum) (ex BitCountry)
- [Moonbeam](#moonbeam)
- [Mythical](#mythical)
- [NFTMozaic](#nftmozaic)
- [Nodle](#nodle)
- [Unique Network](#unique-network)

> ## [Asset Hub](https://wiki.polkadot.network/docs/learn-nft-projects#asset-hub)

Reviewer: ```suggestion
## Asset Hub
```

> The Asset Hub is a generic assets parachain which provides functionality for deploying and transferring assets — both Fungible and Non-Fungible Tokens (NFTs). The Asset Hub currently hosts [Uniques pallet](https://wiki.polkadot.network/docs/learn-nft-pallets#uniques-pallet) (its use on AssetHub is not encouraged) and the [NFTs pallet](https://wiki.polkadot.network/docs/learn-nft-pallets#nfts-pallet).

Reviewer: ```suggestion
The Asset Hub is a [system chain](./learn-system-chains) which provides functionality for deploying and transferring assets — both Fungible and Non-Fungible Tokens (NFTs). The Asset Hub currently hosts [Uniques pallet](./learn-nft-pallets#uniques-pallet) (its use on AssetHub is not encouraged) and the [NFTs pallet](./learn-nft-pallets#nfts-pallet).
```

> A more in-depth overview of the Asset Hub pallets and developer resources available to the developers can be found in the dedicated [wiki pallets section](https://wiki.polkadot.network/docs/learn-nft-pallets).

Reviewer: ```suggestion
A more in-depth overview of the Asset Hub pallets and developer resources available to the developers can be found in the dedicated [section about NFT pallets](./learn-nft-pallets).
```

## Astar

> [Astar Network](https://astar.network/) and its sister network [Shiden Network](https://shiden.astar.network/) are the smart contract infrastructure in the Polkadot Ecosystem. Astar Ecosystem ("Astar") supports NFTs developed with EVM smart contracts and Wasm smart contracts.

Reviewer: ```suggestion
[Astar Network](https://astar.network/) and its sister network [Shiden Network](https://shiden.astar.network/) are smart contract infrastructure in the Polkadot Ecosystem, i.e. they are rollups that support EVM smart contracts. Astar supports NFTs developed with EVM smart contracts and Wasm smart contracts.
```

> Astar has all the toolings available that every EVM NFT developer knows. The availability of those toolings makes the onboarding to Astar networks very attractive to any developer looking to explore the Polkadot Ecosystem. Astar has an active community of artists and NFT enthusiasts. Besides supporting all EVM toolings, Astar also bootstrapped the Wasm smart contract environment for NFT developers writing smart contracts with [ink!](https://use.ink/) based on [PSP34](https://github.com/w3f/PSPs/blob/master/PSPs/psp-34.md) (Polkadot Standards Proposals).

Reviewer: ```suggestion
Astar has all the toolings available that every EVM NFT developer knows. The availability of those toolings makes the onboarding to Astar networks desirable to any developer looking to explore the Polkadot Ecosystem. Astar has an active community of artists and NFT enthusiasts. Besides supporting all EVM toolings, Astar also bootstrapped the Wasm smart contract environment for NFT developers writing smart contracts with [ink!](https://use.ink/) based on [PSP34](https://github.com/w3f/PSPs/blob/master/PSPs/psp-34.md) (Polkadot Standards Proposals).
```

> The main advantage of having a multi-virtual machine environment for NFT developers is that it will give more possibilities to the builders for the use case they are developing. With the support of Wasm smart contracts, developers can develop solutions like [RMRK](https://wiki.polkadot.network/docs/learn-nft#rmrk) with smart contracts.

Reviewer: ```suggestion
The main advantage of having a multi-virtual machine environment for NFT developers is that it gives builders more possibilities for the use cases they are developing. With the support of Wasm smart contracts, developers can create solutions like [RMRK](./learn-nft#rmrk) with smart contracts.
```

- [General information](https://astar.network/blog/An-introduction-to-NFTs-on-Astar-38359)
- [Generating PSP34 NFTs via ink! contracts](https://docs.astar.network/docs/tutorials/from-zero-to-ink-hero/nft/)
- [EVM NFT management example](https://docs.astar.network/docs/build/EVM/astarbase/#example-use-case-discount-price-on-an-nft)

## Basilisk

[Basilisk](https://bsx.fi/) is a Kusama parachain that provides liquidity for the ecosystem. It also has a full-featured NFT platform based on the [Uniques pallet](https://github.com/paritytech/polkadot-sdk/tree/master/substrate/frame/uniques). One of the key features of Basilisk is that it allows minting NFTs with a royalty fee. This royalty fee is distributed to the original creator of the NFT via the runtime pallet. Additionally [Basilisk offers a feature](https://github.com/galacticcouncil/Basilisk-node/tree/master/pallets/marketplace) that allows creating a buy order for a specific NFT.

These NFTs can be viewed and interacted instantly on [KodaDot](https://kodadot.xyz/).

![nft-basilisk](../assets/nft/nft-basilisk.png)

## Crust

Crust is primarily a storage solution. It has a separate section on dedicated NFT storage solutions.

> [Storing NFTs](https://wiki.crust.network/docs/en/buildIntegrationNFTDataStorage)

Reviewer: ```suggestion
Crust is primarily a storage solution. A [separate section](https://wiki.crust.network/docs/en/buildIntegrationNFTDataStorage) is dedicated to NFT storage solutions.
```

## Darwinia Network

> Darwinia Network is a decentralized heterogeneous cross-chain bridge protocol built on Substrate, focusing on decentralized cross-chain token swap, exchange and market. Has an EVM-compatible blockchain.

Reviewer: you mean it is an EVM-compatible chain?

Author: Yes. That is absolutely correct. My phrasing is off. lapsus linguae

No NFT creation documentation available.

> NFT marketplace available at https://nft.io/collection/darwinia/assets

Reviewer: ```suggestion
Darwinia Network is a decentralized heterogeneous cross-chain bridge protocol built on Substrate. It focuses on decentralized cross-chain token swap, exchange, and market and has an EVM-compatible blockchain.

NFT marketplace available [here](https://nft.io/collection/darwinia/assets).
```

## KodaDot

> [KodaDot](https://kodadot.xyz/) is an open-source NFT marketplace that operates on the Dotsama (Kusama/Polkadot) network, striving to aggregate various NFT standards (Kusama, Asset Hub, Basilisk, etc.) in the Dotsama ecosystem, enhancing user experience by abstracting these standards. Its marketplace is currently the main NFT dApp on AssetHub.

Reviewer: ```suggestion
[KodaDot](https://kodadot.xyz/) is an open-source NFT marketplace that operates on the Polkadot Ecosystem and beyond (Polkadot and Kusama Asset Hubs, Mantle, Base, and Immutable zkEVM), striving to aggregate various NFT standards and enhancing user experience.
```

Author: Hats off to you dear sir. That is a mighty good mop up :))))

KodaDot's strength lies in its commitment to open-source collaboration. It has transformed into a collaborative hub where creators, developers, and community members work collectively for decision making, amassing an extensive network of over [90 open-source contributors](https://hello.kodadot.xyz/about-us/who-are-we/achievements). This robust collaboration has earned KodaDot the [number one rank as a dApp in the Polkadot ecosystem on Github](https://github.com/topics/polkadot).

> See below a video tutorial about how to mint your NFT on [the Polkadot Asset Hub](https://wiki.polkadot.network/docs/learn-nft-projects#asset-hub) using KodaDot.

Reviewer: ```suggestion
See below a video tutorial about how to mint your NFT on [the Polkadot Asset Hub](#asset-hub) using KodaDot.
```

  </div>
</div>

For more information about minting using KodaDot see [this step-by-step tutorial](https://hello.kodadot.xyz/tutorial/minting/how-to-mint-nfts-on-polkadot-asset-hub-using-kodadot).

> 

Reviewer: why the removal of this section about the KodaDot Team and Tools?

## MNet Continuum

> MNet Continuum (ex BitCountry) specializes in providing Metaverse as a Service (MaaS) to all networks. The first decentralized application (dApp) offers users the tools to create and customize a metaverse, run and reward a community through immersive quests, and mint and trade NFTs. Users can enjoy metaverse gaming, learning and networking.

Reviewer: ```suggestion
MNet Continuum (ex BitCountry) specializes in providing Metaverse as a Service (MaaS) to all networks. Their decentralized application (dApp) offers users the tools to create and customize a metaverse, run and reward a community through immersive quests, and mint and trade NFTs. Users can enjoy metaverse gaming, learning and networking.
```

Several dozen high profile teams are launching their products (or re-launching) on Moonriver / Moonbeam, however, it is essential to note that Moonbeam is an EVM chain and will therefore suffer from the same limitations as any other EVM chain in regards to customization and feature-richness of NFTs.

A notable advantage, however, is that Moonriver / Moonbeam is still a Substrate chain, meaning integration of custom pallets into the runtime is still possible, making NFT specific optimizations at the chain runtime level a reliable way to keep EVM compatibility of tools while at the same time optimizing storage and interactions for rich NFTs.

A convenient directory of Moonbeam dApps can be found here: [Moonbeam dApps](https://apps.moonbeam.network/moonbeam/app-dir&sa=D&source=docs&ust=1738885283203398&usg=AOvVaw1bnTE-WD_mIwe8VvoUxlmh)

Developer tools and documentation links:

[Minting a Cross-Chain NFT with Axelar SDK](https://docs.moonbeam.network/tutorials/interoperability/using-axelar-sdk/)
[Deploying an ERC-721 Contract Example](https://docs.moonbeam.network/tutorials/eth-api/thirdweb/)

## Mythical

> [Mythical Games](https://mythicalgames.com/) is an online gaming platform that implements a fungible $MYTH token as well as NFTs in its gaming ecosystem. They are a parachain in the Polkadot ecosystem and implement the Polkadot NFTs pallet to provide advanced NFT tokens.

Reviewer: ```suggestion
[Mythical Games](https://mythicalgames.com/) is an online gaming platform that implements NFTs in its gaming ecosystem. Mythical Games has Mythos Chain, an L1 rollup in the Polkadot ecosystem that implements the Polkadot NFTs pallet to provide advanced NFT functionalities.
```

> Mythical offers a platform for game developers with comprehensive documentation and onboarding guides, to ensure anyone can get up and running in time. The primary integration path is REST APIs, but SDKs are also available upon request.
> 
> Developer tools and documentation links:
> 
> [The Mythical Platform API](https://docs.mythicalgames.com/mythical-platform-api)

Reviewer: ```suggestion
Mythical offers a platform for game developers with comprehensive documentation and onboarding guides. The primary integration path is [REST APIs](https://docs.mythicalgames.com/mythical-platform-api), and SDKs are also available upon request.
```

Author: can't commit. status outdated?

- Due to metadata format flexibility and mutability, it is possible and very simple to implement on-chain data obfuscation for enhanced privacy

Unique Network provides a fully featured EVM with a few perks:

- The EVM address space (0x…) is mapped to the corresponding Substrate address space which simplifies object and contract access between the EVM and Substrate objects via CrossAddressing mechanism. This means that you can mix and match technologies. For example you can have a native NFT call an EVM contract. This is the core mechanism for extending and customising NFTs for specific use cases.
- This also means that a native Substrate object is addressable in EVM applications without an intermediary address handling contract, i.e. applications like the Metamask wallet can address Unique Network Substrate objects directly. This greatly improves efficiency.
- Unique network EVM utilities contain interfaces to the blockchain’s core Substrate pallet functions which allows Solidity contracts to directly invoke them.
- This interoperability is fully supported by the JS/TS SDK allowing rapid development even in the mix-and-match scenario.

> The recommended and most efficient way to build on Unique Network is via the JS/TS SDK version 2.0 at the moment of this writing. The SDK interacts with the blockchain via an RPC mechanism eliminating the need to build and maintain a complex development environment. It fully implements the [Unique Metadata Schema 2.0](https://docs.unique.network/reference/schemas/). The Unique SDK has recently incorporated the ability to create NFT assets on the Asset Hub as well which makes it a very versatile tool for NFT management and creation on the Polkadot ecosystem.

Reviewer: ```suggestion
The recommended and most efficient way to build on Unique Network is via the JS/TS SDK version 2.0. The SDK interacts with the blockchain via an RPC eliminating the need to build and maintain a complex development environment. It fully implements the [Unique Metadata Schema 2.0](https://docs.unique.network/reference/schemas/). The Unique SDK allows the creation of NFT assets on the Asset Hub as well which makes it a very versatile tool for NFT management and creation on the Polkadot ecosystem.
```

Developer tools and documentation links:

- [General information](https://docs.unique.network/about)
- [Substrate native advanced NFTs (NFT 2.0)](https://docs.unique.network/build/sdk/v2/quick-start.html) 
- [Building NFTs in an EVM](https://docs.unique.network/build/evm/)
- [Tutorials/Cookbook](https://docs.unique.network/tutorials)  
- [Youtube Workshops and Tutorials](https://www.youtube.com/watch?v=StfRap-dvks&list=PLFSU2-RmX858vLu9a5bukOTpJ1dYzsE4o)
- [Reference Documentation](https://docs.unique.network/reference)
- [NFT Marketplace](https://unqnft.io/market?filterState=)
- [Indexer/Block Explorer](https://uniquescan.io/unique/)

> The most prominent projects that currently implement Unique Network’s advanced NFT capabilities are

Reviewer: What about Sovereign Nature?

Author: The project has officially ceased operation.
